framework/src/utils/SlepcSupport.C

```diff
@@ -122,29 +129,28 @@ setSlepcEigenSolverTolerances(EigenProblem & eigen_problem, const InputParameter
   if (eigen_problem.isNonlinearEigenvalueSolver())
   {
     // nonlinear solver tolerances
-    Moose::PetscSupport::setSinglePetscOption("-eps_power_snes_max_it",
+    Moose::PetscSupport::setSinglePetscOption("-snes_max_it",
                                               stringify(params.get<unsigned int>("nl_max_its")));
 
-    Moose::PetscSupport::setSinglePetscOption("-eps_power_snes_max_funcs",
+    Moose::PetscSupport::setSinglePetscOption("-snes_max_funcs",
                                               stringify(params.get<unsigned int>("nl_max_funcs")));
 
-    Moose::PetscSupport::setSinglePetscOption("-eps_power_snes_atol",
+    Moose::PetscSupport::setSinglePetscOption("-snes_atol",
                                               stringify(params.get<Real>("nl_abs_tol")));
 
-    Moose::PetscSupport::setSinglePetscOption("-eps_power_snes_rtol",
+    Moose::PetscSupport::setSinglePetscOption("-snes_rtol",
                                               stringify(params.get<Real>("nl_rel_tol")));
 
-    Moose::PetscSupport::setSinglePetscOption("-eps_power_snes_stol",
+    Moose::PetscSupport::setSinglePetscOption("-snes_stol",
                                               stringify(params.get<Real>("nl_rel_step_tol")));
 
     // linear solver
-    Moose::PetscSupport::setSinglePetscOption("-eps_power_ksp_max_it",
+    Moose::PetscSupport::setSinglePetscOption("-ksp_max_it",
                                               stringify(params.get<unsigned int>("l_max_its")));
 
-    Moose::PetscSupport::setSinglePetscOption("-eps_power_ksp_rtol",
-                                              stringify(params.get<Real>("l_tol")));
+    Moose::PetscSupport::setSinglePetscOption("-ksp_rtol", stringify(params.get<Real>("l_tol")));
 
-    Moose::PetscSupport::setSinglePetscOption("-eps_power_ksp_atol",
+    Moose::PetscSupport::setSinglePetscOption("-ksp_atol",
```

Comment:
This looks like a really nice change

Reply:
haha. Get rid of prefix because nobody like it